### ISSUE TITLE
Cannot get text to display

### ISSUE THREAD

**Issue by Beebolini** [White-Tiger/T-Clock#51]
Hello!

Installed 2.4.1 build 419 on two different Win10 Anniversary update computers.  No probs on one of them, but the other isn't displaying the clock text.  Settings are the same on both computers.  Both computers had an old version of T-Clock installed, and both installs popped the screen about converting settings to the newer version.

This is all I see on the problem computer:
![screen shot](https://cloud.githubusercontent.com/assets/4278319/17578136/5f3ceed0-5f53-11e6-8c56-7a9eae72a92b.png)

Everything else seems to be working fine.  I've tried changing fonts, font and background colours, and a number of other settings.  Rebooting hasn't helped, either.

This could very well be a display driver issue.  It's a six-year-old Alienware.

Is there a way to copy settings from one computer to the other, so I'm certain they are exactly the same?

Happy to provide any additional information you need.  Thanks very much for maintaining this awesome software!  It made me very happy when I found it, after the Anniversary update broke the old version.

Also thanks in advance for any assistance!

**Comment by Beebolini** [comment]
All righty.  I played around for about two hours (off and on) before writing that post.  Five minutes after writing it, I opened the Options page again, and immediately noticed that "Line Spacing" was set to 64 for some reason.  Dropped it to 2, and voila - instantly awarded myself the Dumbass of the Day™ prize.

I'll leave the post here in case anyone needs a laugh.  In the meantime, I'm going to go drink.

Huge thanks again for continuing this project!

**Comment by White-Tiger** [comment]
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

What have you done :/
This kind of "bug" has been reported at least twice _(now thrice)_ but all the time, people ended up playing around or deleting their settings...
What I _need_ is the bugged out configuration with all settings intact... so I can figure out what went wrong and adjust the settings converter to fix it automatically...

Though your note on "Line Spacing" = 64 is at least a hint... as I can try that with Stoic Joker's T-Clock and see how it behaves... Anyway, do you remember the exact version you've been using before?

**Comment by Beebolini** [comment]
Very sorry.  I like to think I'm a capable user, but never having coded before, my brain's not totally in the mindset it needs to be to help you out properly.  I tried really hard to replicate the issue by uninstalling and trying to go back to an old version, but no success.

I've been using different versions of the program on this computer for a very long time.  The latest version I had installed would have been Stoic Joker's build 95.  I still have the install file.
[T-Clock 2010 (build 95).zip](https://github.com/White-Tiger/T-Clock/files/413701/T-Clock.2010.build.95.zip)

That's the only version that would ever have been installed on the computer that did not have an issue.  the computer that DID have an issue is the six year old Alienware I mentioned above, and that's the one that would have had at least a few versions of the software installed over the years.  Perhaps there's multiple settings sitting around that interfered with each other?

The problem happened right away - I was never able to see the clock text on the new version, until I discovered the Line Spacing setting.

I sincerely hope that at least helps a bit!

Have a great day,

Beebolini

**Comment by White-Tiger** [comment]
well, the "Line Spacing" hint helped quite a bit :P It's the changes I've made in https://github.com/White-Tiger/T-Clock/commit/e4406d015cb9b4847865624f49d2790473011867#diff-17d5222e7434c09f9c404d9d5b97b4ceL125
T-Clock read 32bit values, but casted them to 16bit _(thus skipping the higher 16bits)_ which is an issue when we're using signed _(negative)_ values... and removing those casts screwed it up a bit...

Which means I've got the picture now ;) Thanks to you and your time investment on your hunt for the corrupt setting.
Sorry for the trouble caused. You're hopefully the last to have this issue.
